python/lsst/ts/externalscripts/auxtel/latiss_base_align.py

```diff
@@ -741,25 +746,29 @@ async def arun(self, checkpoint: bool = False) -> None:
         self.offset_total_focus = 0.0
         self.offset_total_coma_x = 0.0
         self.offset_total_coma_y = 0.0
+
+        # Getting initial hexapod position
+        self.initial_hexapod_position = await self.latiss.get_hexapod_position()
```

Comment:
The `LATISS` class deals only with the camera, it doesn't have this method defined in it. This would most likely be a method in the `ATCS` class, which also doesn't have such a method. 

I think you can either create it as part of this ticket, or implement something here that will get it for you. Either way works fine, however, this might be a good addition to the `ATCS` class so if you prefer to implement it here, we should probably have a ticket to port it to ts_observatory_control.

Reply:
Lesson learned: don't trust autocompletion. What a silly mistake :(  

I think that using the `atcs.offset_aos_lut` method in the way I'm proposing below would eliminate the need to have a
`atcs.get_hexapod_position()` method here. In any case, it would be useful to implement it on another ticket. 

Please, let me know your thoughts on this.